server/controllers.js

```diff
@@ -1,24 +1,48 @@
-const db = require('./db/db-config');
-const rp = require('request-promise');
+const db = require('./db/db-config.js');
+const dbHelpers = require('./db/helpers.js');
+const Promise = require('bluebird');
 
+const createPromise = Promise.promisify(dbHelpers.create);
+const retrievePromise = Promise.promisify(dbHelpers.retrieve);
+const updatePromise = Promise.promisify(dbHelpers.update);
+const deletePromise = Promise.promisify(dbHelpers.delete);
 
 // USERS -------------------------------->
-exports.getAllUserData = (req, res) => {
-  console.log('getuserdata');
-  res.send();
-
+exports.getUserData = (req, res) => {
+  let userId = 101; // LATER - get from front end? Session? 
```

Comment:
If you end up using tokens, you can store the userID in the token that you send to the client, and retrieve it off of the token (on the `req` object) whenever you need it.  If you use just sessions, you'll need to manage all of this on the back-end.